src/index.ts

```diff
@@ -200,11 +212,8 @@ class Monika extends Command {
       startPrometheusMetricsServer(flags.prometheus)
     }
 
-    if (flags['create-config']) {
-      log.info(
-        'Opening Monika Configuration Generator in your default browser...'
-      )
-      await open('https://hyperjumptech.github.io/monika-config-generator/')
+    const isOpenConfigGenPage = await this.openConfigGenerator(flags)
```

Comment:
If the variable name start with `is`, I would expect that variable will give me boolean value without side effect. Looks like the function `openConfigGenerator` has side effect (`open('https://hyperjumptech.github.io/monika-config-generator/')`).

Reply:
fixed.